src/fetch/orderbook.py

```diff
@@ -97,6 +97,7 @@ def get_batch_rewards(cls, block_range: BlockRange) -> DataFrame:
             open_query("orderbook/batch_rewards.sql")
             .replace("{{start_block}}", str(block_range.block_from))
             .replace("{{end_block}}", str(block_range.block_to))
+            .replace("{{EPSILON}}", "10000000000000000")  # ETH cap for payment (in WEI)
```

Comment:
Not sure if we go that way, but since the new CIP for adjusting and renewing the solver rewards contains an asymmetric cap, we should actually introduce two parameters here.

Reply:
I mostly introduced the parameter for consistency with the queries in the solver rewards repo.

If we go for an asymmetric cap, we should add that in a separate PR, imo.